app/index.html

```diff
@@ -54,7 +54,8 @@
             <span ng-if="event.venue">@{{ event.venue }}</span>
           </span>
           <p ng-if="event.description" class="description">{{ event.description }}</p>
-          <p ng-if="event.distanceToUser" class="distance description">{{ event.distanceToUser | distanceFormat}} km</p>
+          <p ng-if="event.distanceToUser <= 1" class="distance description">{{ event.distanceToUser | distanceFormat : "m"}}</p>
+          <p ng-if="event.distanceToUser > 1" class="distance description">{{ event.distanceToUser | distanceFormat : "km"}}</p>
```

Comment:
Can the distanceFormat be set with ?:-operator (distanceToUser > 1 : "km" : "m")? That would cut the need for two p elements doing the same thing.